### ISSUE TITLE
AOD problem on 8.2.8 version

### ISSUE THREAD

**Issue by Aero-144** [j-kaltes/Juggluco#216]
Hi!
After installation 8.2.8 app version measurement on AOD (always on display notification) is broken. It seems that place for measurement used, but it doesn't visible (other apps icons shifted). Usual notification works fine.

**Comment by j-kaltes** [comment]
If  you are using Libre 2, you can try the no3 version to see if it has the same problem.

**Comment by Aero-144** [comment]
I use Sibionics. Is it important what kind of sensor is used?  On 8.2.2 version it works, but I have problem with color notification when dark/light mode theme changed. If I unchecked Glucose notification in settings menu, I see on AOD icon of juggluco now. If I enable this option measurement becomes invisible.

**Comment by j-kaltes** [comment]
No, important is the Juggluco version. Sibionics can't be used with the no3 version.
So Juggluco still works with some Sibionics sensors?

**Comment by Aero-144** [comment]
Yes. It works fine. But what do you mean about no3 version? What version is it?

**Comment by AgushkaNightscout** [comment]
> Yes. It works fine. But what do you mean about no3 version? What version is it?

It's arm32 version without FSL3 support

**Comment by Aero-144** [comment]
> > Yes. It works fine. But what do you mean about no3 version? What version is it?
> 
> It's arm32 version without FSL3 support

I understand. Thanks. No, I use arm64 only version of the app.

**Comment by j-kaltes** [comment]
There is also a arm64 version of no3. It is meant for devices running Android 7.1 and lower and because of that Libre3 isn't included. Most of them are arm32.
Because icons created in the app itself can't be used in older version of Android, Juggluco included already created icons for the glucose level displayed in the Android status bar. For higher version of Android that is not really needed, so I removed them from Juggluco 8.2.7+ for Android 8 and higher. This is a change in the notifications, maybe that some how created your problems, but I don't see how.
![sizes](https://github.com/user-attachments/assets/600cb8d0-5d70-4413-9e71-f8d9bacf653a)

**Comment by Aero-144** [comment]
Can I help something to localize the problem?
